src/QirRuntime/build-qir-runtime.ps1

```diff
@@ -1,34 +1,51 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
build.py uses slightly different folder structure to avoid conflicts between win and wsl builds on the same machine. I can also run it from cmd rather than from PS which I find convenient (and linux users might prefer it to PS as well). it's for local development only and is more flexible if you'd like to do custom builds locally (e.g. easy to do just make)